src/sentry/models/debugfile.py

```diff
@@ -361,6 +362,23 @@ def _analyze_progard_filename(filename: str) -> Optional[str]:
         return None
 
 
+@region_silo_only_model
+class ProguardArtifactRelease(Model):  # type: ignore
+    __include_in_export__ = False
+
+    organization_id = BoundedBigIntegerField(db_index=True)
+    project_id = BoundedBigIntegerField(db_index=True)
```

Comment:
While it can be done in a future pull request. It would be good to have this model added to [project deletions](https://github.com/getsentry/sentry/blob/master/src/sentry/deletions/defaults/project.py) so that when customers remove a project proguard metadata is also removed.